services/web/src/lib/reusable-components/dropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import { Select, SelectContent, SelectGroup, SelectItem, SelectTrigger, SelectValue } from '@/components/ui/select';
+import { FormField, FormItem, FormLabel, FormMessage } from '@/components/ui/form';
+
+type dropdownInput = {
+    name: string;
+    value: string;
+};
+
+type DropdownComponentProps = {
+    label: string;
+    placeholder: string;
+    className: string;
+    items: dropdownInput[];
+};
+
+export function DropdownComponent({ label, placeholder, className, items }: DropdownComponentProps) {
```

Comment:
You will have to make the component receive classnames for everything that can be customised.